assets/images/part-5/DDIRatio.csv

```diff
@@ -0,0 +1,15 @@
+ID,Study ID,Reference,Grouping,Victim,Perpetrator,Route Victim,Route Perpetrator,Compartment,AUCR Avg,AUCR AvgType,AUCR Var,AUCR VarType,CmaxR Avg,CmaxR AvgType,CmaxR Var,CmaxR VarType,t_placebo_0,t_placebo_end,t_treatment_0,t_treatment_end,t Unit,Comment,Dose,Dose free API,Dose Unit,Route,Times of Administration,Times Unit,Comment Regimen,Administered form,Formulation type,Infusion duration [min],Water volume ingested for drug intake [mL],Comment on administration/formulation,Fasted/Fed state,Duration of fasting before drug administration [h],Duration of fasting after drug administration [h],Comment on food intake,Notghi
```

Comment:
Because we don't have any example of the DDIRatio table on the OSP, I've added it temporarily here.